app/services/reporting/account_deletion_rate_report.rb

```diff
@@ -13,6 +13,18 @@ def account_deletion_report
       table
     end
 
+    def account_deletion_emailable_report
+      EmailableReport.new(
+        email_options: {
+          title: 'Account deletion rate (last 30 days)',
+          float_as_percent: true,
+          precision: 4,
+        },
+        table: account_deletion_report,
+        csv_name: 'account_deletion_rate',
+      )
+    end
```

Comment:
@ThatSpaceGuy In the event my PR lands before yours, we made some changes after you dropped form pairing, as reflected here.

We moved the `Struct` out into its own `Reporting::EmailableReport` "class", and got rid of the `metadata` stuff. It's all returned inline.